homeassistant/components/agent_dvr/camera.py

```diff
@@ -117,6 +119,11 @@ async def async_update(self):
             "alerts_enabled": self.device.alerts_active,
         }
 
+    @property
+    def should_poll(self) -> bool:
+        """Update the state periodically."""
+        return True
```

Comment:
How is this change related to the camera state?

Reply:
The camera does not update without this. Adding this back in ensures it polls and attributes are then retrieved, solving the user's issue. I've tested my agent the best I can and everything works now. @rotem925 should reply back soon confirming this.

Reply:
Yep, that's fine now...

Reply:
https://github.com/home-assistant/core/issues/54028#issuecomment-893519529